scripts/chains/smingot_chain.zs

```diff
@@ -0,0 +1,328 @@
+	.buildAndRegister();
+	
+cutting_saw.recipeBuilder()
+	.inputs(<ore:plateDenseMatterGrahamHot>)
+	.outputs(<ore:plateMatterGrahamHot>.firstItem * 9)
+	.EUt(60)
+	.duration(200)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:dustCocoa>)
+	.fluidOutputs(<liquid:unsweetened_chocolate> * 144)
+	.EUt(180)
+	.duration(20)
+	.buildAndRegister();
+
+macerator.findRecipe(2, [<minecraft:dye:3>], []).remove();
+macerator.recipeBuilder()
+	.inputs(<ore:plateDensePressCakeGraded>)
+	.outputs(<ore:dustCocoa>.firstItem)
+	.EUt(120)
+	.duration(120)
+	.buildAndRegister();
+	
+mixer.recipeBuilder()
+	.inputs(<nuclearcraft:gelatin> * 2)
+	.fluidInputs(<liquid:albumen> * 500)
+	.fluidInputs(<liquid:sweetened_diluted_cane_syrup_mixture> * 5000)
+	.fluidOutputs(<liquid:marshmallow_syrup_mixture> * 5000)
+	.EUt(120)
+	.duration(300)
+	.buildAndRegister();
+
+assembler.recipeBuilder()
+	.inputs(<ore:plateDensePressCake>)
+	.inputs(sensors[3])
+	.chancedOutput(<ore:plateDensePressCakeGraded>.firstItem, 5000, 100)
+	.EUt(480)
+	.duration(40)
+	.buildAndRegister();
+	
+assembler.recipeBuilder()
+	.inputs(<ore:plateDensePressCakeDutched>)
+	.inputs(sensors[3])
+	.chancedOutput(<ore:plateDensePressCakeGraded>.firstItem, 7500, 300)
+	.EUt(480)
+	.duration(40)
+	.buildAndRegister();
+	
+metal_bender.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedMatterGrahamHot>)
+	.property("circuit", 0)
+	.outputs(<ore:plateDenseMatterGrahamHot>.firstItem)
+	.EUt(45)
+	.duration(80)
+	.buildAndRegister();
+	
+centrifuge.recipeBuilder()
+	.inputs(<ore:egg>)
+	.fluidOutputs(<liquid:albumen> * 100)
+	.fluidOutputs(<liquid:yolk> * 100)
+	.EUt(45)
+	.duration(60)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorPressed> * 2)
+	.outputs(<ore:plateDensePressCake>.firstItem)
+	.fluidOutputs(<liquid:cocoa_butter> * 144)
+	.EUt(120)
+	.duration(100)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorDutchedPressed> * 2)
+	.outputs(<ore:plateDensePressCakeDutched>.firstItem)
+	.fluidOutputs(<liquid:cocoa_butter> * 144)
+	.EUt(120)
+	.duration(100)
+	.buildAndRegister();
+	
+furnace.addRecipe(<ore:crushedCentrifugedMatterGrahamHot>.firstItem, <ore:crushedCentrifugedMatterGraham>);
+
+var gelatins as IIngredient[] = [<ore:stickBone>, <ore:leather>, <minecraft:porkchop>, <minecraft:beef>, <minecraft:chicken>, <minecraft:rabbit>, <minecraft:mutton>];
+
+for gelatin_producer in gelatins {
+	extractor.recipeBuilder()
+		.inputs(gelatin_producer)
+		.outputs(<nuclearcraft:gelatin>)
+		.EUt(120)
+		.duration(100)
+		.buildAndRegister();
+}
+
+fluid_solidifier.recipeBuilder()
+	.fluidInputs(<liquid:fish_oil> * 400)
+	.inputs(<metaitem:shape.mold.plate>)
+	.outputs(<nuclearcraft:gelatin>)
+	.EUt(60)
+	.duration(100)
+	.buildAndRegister();
+	
+mixer.recipeBuilder()
+	.inputs(<ore:dustSugar>)
+	.inputs(<ore:dustWheat> * 3)
+	.inputs(<ore:dustSodiumBicarbonate>)
```

Comment:
You don't adhere to mold conventions here. It should be 6 dust, since Sodium Bicarbonate is NaHCO3. Dust count is the number of components in the item.

Reply:
Still needs work here. Sugar is 1 dust, and wheat is 3. Sodium Bicarbonate is 6 dust, and it should produce 10 dust of output, since 1 + 3 + 6 = 10.